pages/expense.js

```diff
@@ -90,6 +98,18 @@ const expensePageQuery = gqlV2/* GraphQL */ `
             valueInCents
             currency
           }
+          totalPaidExpenseInvoices: totalPaidExpense(expenseType: [INVOICE], dateFrom: $totalPaidExpenseDateFrom) {
```

Comment:
I would ask the value to be converted in the currency of the Collective or Host paying the expense. `currency: $collectiveCurrency`